client/src/shared/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+
+export interface collapseProps {
+  label: string
+  subLabel?: string
+  children: React.ReactNode
+};
+
+const Collapse: React.FC<collapseProps> = ({
+  label = 'Install Xampp',
+  subLabel = 'Version 1 of 20',
+  children
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleCollapse = (): void => {
+    setIsOpen(!isOpen);
+  };
+
+  const arrowClasses = isOpen ? 'rotate-180 order-last' : 'order-last';
+
+  return (
+    <div>
+      <div>
+        <div
+          className='bg-neutral-100 pb-2 pt-3 pb-4 rounded relative flex place-items-center justify-between'
+          onClick={toggleCollapse}
+        >
+          <svg
+            xmlns='http://www.w3.org/2000/svg'
+            fill='none'
+            viewBox='0 0 24 24'
+            stroke-width='1.5'
+            stroke='currentColor'
+            className='w-6 h-6 absolute ml-4 mt-1'
+          >
+            <path
+              stroke-linecap='round'
+              stroke-linejoin='round'
+              d='M3.75 5.25h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5m-16.5 4.5h16.5'
+            />
+          </svg>
```

Comment:
- I believe we already have a menu icon for this
- You can import the menu icon found in the icons folder so that you can use it in this component